### PR TITLE
fix ~system/ auto import for vscode

### PR DESCRIPTION
**Pull request by gonpombo8**
Issue https://github.com/decentraland/sdk/issues/567

### CODE REVIEW

**Comment by menduz** [comment]
WhY/HOW does this black magic work?

File: packages/@dcl/sdk/types/tsconfig.ecs7.json [hunk 1]
```diff
@@ -29,7 +29,23 @@
       "ES2020"
     ],
     "types": [
-      "@dcl/js-runtime",
+      "~system/CommunicationsController",
```
Comment (gonpombo8) [inline packages/@dcl/sdk/types/tsconfig.ecs7.json:32]: yes i know, but they are auto generated

File: packages/@dcl/sdk/types/tsconfig.ecs7.json [hunk 2]
```diff
@@ -29,7 +29,25 @@
...
+      "~system/Runtime",
+      "~system/Scene",
+      "~system/SignedFetch",
+      "~system/Testing",
+      "~system/UserActionModule",
+      "~system/UserIdentity",
+      "@dcl/js-runtime"
+    ],
+    "typeRoots": [
+      ".",
```
Comment (gonpombo8) [inline packages/@dcl/sdk/types/tsconfig.ecs7.json:48]: import all the ~system from '.'

File: packages/@dcl/sdk/types/tsconfig.ecs7.json [hunk 3]
```diff
@@ -29,7 +29,25 @@
...
+      "../..",
```
Comment (gonpombo8) [inline packages/@dcl/sdk/types/tsconfig.ecs7.json:49]: import the @dcl/js-runtime from '../../

**Pull request closed without merging**